Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Identity_Analytics.yml

```diff
@@ -0,0 +1,1775 @@
+      - "18"
+    scriptarguments:
+      verdict:
+        simple: Malicious
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1890
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    fieldMapping:
+    - incidentfield: User Risk Level
+      output:
+        simple: ${PaloAltoNetworksXDR.RiskyUser.risk_level}
+    - incidentfield: Failed Logon Events
+      output:
+        complex:
+          root: AzureFailLoginCount
+          transformers:
+          - operator: append
+            args:
+              item:
+                value:
+                  simple: NumOfOktaFailedLogon
+                iscontext: true
+    - incidentfield: Email
+      output:
+        simple: ${ActiveDirectory.Users.mail}
+    - incidentfield: Account Member Of
+      output:
+        simple: ${ActiveDirectory.Users.memberOf}
+    - incidentfield: Account Status
+      output:
+        simple: ${Account.Status}
+    - incidentfield: Cloud Account ID
+      output:
+        simple: ${MSGraphUser.ID}
+    - incidentfield: Account ID
+      output:
+        complex:
+          root: Account
+          filters:
+          - - operator: notContainsGeneral
+              left:
+                value:
+                  simple: Account.ID
+                iscontext: true
+              right:
+                value:
+                  simple: "="
+          - - operator: isEqualString
+              left:
+                value:
+                  simple: Account.Type
+                iscontext: true
+              right:
+                value:
+                  simple: Okta
+              ignorecase: true
+          accessor: ID
+    - incidentfield: Manager Email Address
+      output:
+        simple: ${UserManagerEmail}
+    - incidentfield: Alert Name
+      output:
+        simple: ${inputs.AlertName}
+    - incidentfield: Detected User
+      output:
+        simple: ${inputs.Username}
+    - incidentfield: Username
+      output:
+        simple: ${inputs.Username}
+    - incidentfield: XDR Alert Search Results
+      output:
+        simple: ${PaloAltoNetworksXDR.Alert}
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "18":
+    id: "18"
+    taskid: 6bd47335-240d-4bba-88e9-0eefce8b8aea
+    type: title
+    task:
+      id: 6bd47335-240d-4bba-88e9-0eefce8b8aea
+      version: -1
+      name: Remediation
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "19"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 2050
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "19":
```

Comment:
The Else flow will complete the playbook without checking if the user remediated the threat manually. Is that the intended behavior?

Reply:
Done